sdk/keyvault/Azure.Security.KeyVault.Keys/tests/samples/Sample4_EncryptDecrypt.cs

```diff
@@ -53,7 +53,6 @@ public void EncryptDecryptSync()
             Debug.WriteLine($"Decrypted data using the algorithm {decryptResult.Algorithm}, with key {decryptResult.KeyId}. The resulting decrypted data is {Encoding.UTF8.GetString(decryptResult.Plaintext)}");
             #endregion
 
-            #region Snippet:KeysSample4DeleteKey
```

Comment:
How come this region marker got removed?

Reply:
They weren't used. @pakrym made a change a week or two ago to err on unused snippets in case it was unintentional.